x-pack/platform/plugins/shared/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/field_components/xjson_editor.tsx

```diff
@@ -14,21 +14,24 @@ import { TextEditor } from './text_editor';
 interface Props {
   field: FieldHook<string>;
   editorProps: { [key: string]: any };
+  euiFieldProps?: Record<string, any>;
+  [key: string]: any;
```

Comment:
why is `[key: string]: any;` needed? It would be nice to avoid such loose types.

Same for `euiFieldProps?: Record<string, any>;`